static/app/views/settings/organizationMembers/organizationMembersList.tsx

```diff
@@ -27,21 +30,32 @@ import {
 import EmptyMessage from 'app/views/settings/components/emptyMessage';
 
 import MembersFilter from './components/membersFilter';
+import InviteRequestRow from './inviteRequestRow';
 import OrganizationMemberRow from './organizationMemberRow';
 
+const StyledPanelItem = styled('div')`
```

Comment:
@Neo-Zhixing for whatever reason, the convention we have at Sentry is to put styled component at the bottom of the filee

Reply:
thanks for the heads up! component moved.